src/types/ibc.ts

```diff
@@ -1,28 +1,31 @@
 import { codec } from "@cosmjs/stargate";
 
-const connection = codec.ibc.core.connection.v1;
-const channel = codec.ibc.core.channel.v1;
-
 // IBC common types
-export const IbcConnectionState = connection.State;
-export const IbcChannelState = channel.State;
-export const IbcOrder = channel.Order;
+export type IbcConnectionState = codec.ibc.core.connection.v1.State;
+export const IbcConnectionState = codec.ibc.core.connection.v1.State;
```

Comment:
This is pretty weird. Can you not just `export { codec.ibc.core.connection.v1.State as IbcConnectionState }`?

Reply:
It's weird, but I tried these options and none of them worked, some fail in that file and others fail in `src/utils/ibc.ts`:

1.
```typescript
export { codec.ibc.core.connection.v1.State as IbcConnectionState }
```

2.
```typescript
const placeholder = codec.ibc.core.connection.v1.State;
export { placeholder as IbcConnectionState };
```

3.
```typescript
export type { codec.ibc.core.connection.v1.State as IbcConnectionState }
```

4.
```typescript
type placeholder = codec.ibc.core.connection.v1.State;
export type { placeholder as IbcConnectionState };
```

5.
```typescript
const placeholder = codec.ibc.core.connection.v1.State;
export type { placeholder as IbcConnectionState };
```

6.
```typescript
type placeholder = codec.ibc.core.connection.v1.State;
export { placeholder as IbcConnectionState };
```

Reply:
OK, I hadn't realised you couldn't export a nested name when using `as`, and since we need both the type and the variable I guess this is how to do it.